src/python/grpcio_channelz/.gitignore

```diff
@@ -1,4 +1,3 @@
-*.proto
```

Comment:
This is here because we copy from `/proto/grpc/channelz/channelz.proto` to here with [the `setup.py preprocess`](https://github.com/grpc/grpc/blob/4a81ed215b5e8e84a5c818e41e1d641981f574bc/src/python/grpcio_channelz/channelz_commands.py#L28) command. Theoretically, we don't actually need this command anymore and are free to turn it into a no-op. This can be as a follow-up (backlog item).

Reply:
Ah, I didn't know this, I'll add some `TODOs` to change them to no-op.